sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2023-2024.bpmn

```diff
@@ -766,7 +773,7 @@
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=if (calculatedDataTotalFamilyIncome &#60;= dmnPartTimeAwardFamilySizeVariables.limitAwardCSPTIncomeCap)&#10;then dmnPartTimeAwardAllowableLimits.limitAwardCSPTAmount&#10;else&#10;  max(dmnPartTimeAwardAllowableLimits.limitAwardCSPTAmount - ((calculatedDataTotalFamilyIncome - dmnPartTimeAwardFamilySizeVariables.limitAwardCSPTIncomeCap) * dmnPartTimeAwardFamilySizeVariables.limitAwardCSPTSlope),&#10;     100&#10;  )" target="federalAwardCSPTAmount" />
-          <zeebe:output source="=if (awardEligibilityCSPT = true and calculatedDataTotalRemainingNeed1 &#62;= 100)&#10;then&#10;  min(&#10;     calculatedDataTotalRemainingNeed1,&#10;     min(dmnPartTimeAwardAllowableLimits.limitAwardCSPTAmount, federalAwardCSPTAmount)&#10;  )&#10;else&#10;0" target="federalAwardNetCSPTAmount" />
+          <zeebe:output source="=if (awardEligibilityCSPT = true and calculatedDataTotalRemainingNeed1 &#62;= 100)&#10;then&#10;  min(&#10;     calculatedDataTotalRemainingNeed1,&#10;     min(dmnPartTimeAwardAllowableLimits.limitAwardCSPTAmount - programYearTotalPartTimeCSPT, federalAwardCSPTAmount)&#10;  )&#10;else&#10;0" target="federalAwardNetCSPTAmount" />
```

Comment:
In case when the highlighted part is less than 100, the amount will then be awarded.

Is that not against rule of CSPT award should be minim 100. ? Or am I missing something here.?

![image](https://github.com/bcgov/SIMS/assets/54600590/e93abc59-bb36-4fcf-b969-a2b0d5bfdab7)

Reply:
@dheepak-aot I believe that the question would be valid.
I will leave it to @CarlyCotton for confirmation.

Reply:
@dheepak-aot @andrewsignori-aot 
Award should be lesser of: remaining need, PY limit - PY consumed, federalAwardCSPTAmount
Unless it's <100, then it's 0.

Reply:
@guru-aot I would recommend to simplify the logic this. 

![image](https://github.com/bcgov/SIMS/assets/54600590/ff17b73f-265d-4bc8-ac49-4d43b9c41043)

Reply:
Is that a suggestion or a blocker? Because I believe both types are self explanatory for any one who is looking into it. Please explain does it need to be refactored?

Reply:
As the code works, I wouldn't call that as exactly a blocker but a high recommendation as existing code checks for `>=100` in 2 different variables.